.github/workflows/check.yml

```diff
@@ -0,0 +1,20 @@
+name: code-style-check
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
```

Comment:
@Adman should we pin a specific version here? Like the version we've let everyone know we'll be using?

Reply:
Yeah, pinned to 3.8.